src/Entities/Entity.cpp

```diff
@@ -2285,7 +2285,8 @@ void cEntity::WriteMetadata(cMetadataWriter & a_Writer) const
 	a_Writer.SkipMeta();  // Is silent
 	if (a_Writer.m_ProtocolVersion >= PROTO_VERSION_1_10_0)
 	{
-		a_Writer.WriteBool(GetGravity() == 0);  // No gravity
+		const double EPS = 0.0000001;
```

Comment:
The constant is also defined in `SetYawFromSpeed` and `SetPitchFromSpeed`, which is why I copied it again.

I think it might actually be better to not use an epsilon from 0 and instead check if it's changed from the default gravity (since NO_GRAVITY means the clientside gravity code isn't applied).  I think a `DEFAULT_GRAVITY` constant would work well for that.

Working on this...